src/main/scala/com/fulcrumgenomics/util/PhredValue.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.fulcrumgenomics.util
+
+import dagr.commons.CommonsDef._
+
+/** Methods and values to help convert to and from Phred-scaled probabilities.
+  *
+  * A number of implicit conversions are defined in this object.  Integers are implicitly are interpreted as
+  * literal phred values, while Floats and Doubles converted to Phred values using `toPhred`.
+  */
+object PhredValue {
+  /** Phred-scaled zero probability. */
+  val ZeroProbability = PhredValue(Double.PositiveInfinity)
+  /** Phred-scaled probability of one. */
+  val OneProbability  = PhredValue(0.0)
+  /** Phred value of 0. */
+  val ZeroPhred       = PhredValue(0.0)
+  /** The precision to use when converting to an Integer. */
+  val Precision       = 0.0000000001 // Q100
+  /** converts a probability to a phred score, with no validation. */
+  def toPhred(pr: Double) = -10.0*Math.log10(pr)
+  /** converts to a probability from a phred score, with no validation. */
+  def toPr(phred: Double) = Math.pow(10.0, phred / -10.0)
+  /** Gets the mean of the given phred values. */
+  def mean(phred: PhredValue*): PhredValue = phred.foldLeft(ZeroProbability)((sum, qual) => sum + qual) / phred.length.toDouble
+  /*
+  implicit def phredValueWrapper[T](v: T)(implicit numeric: Numeric[T]): PhredValue = new PhredValue(value=numeric.toDouble(v))
+  implicit class NumericPhredValue[T](v: T)(implicit numeric: Numeric[T]) {
+    def toPhredValue: PhredValue = new PhredValue(value=numeric.toDouble(v))
+  }
+  */
+  implicit def intToPhredWrapper(v: Int): PhredValue = new PhredValue(value=v)
```

Comment:
I really don't like these implicit conversions for a number of reasons.  Do you really need them?